quantized_mesh_tile/terrain.py

```diff
@@ -180,6 +180,8 @@ class TerrainTile(object):
     ])
 
     BYTESPLIT = 65636
+    MIN = 0.0
+    MAX = 32767.0
```

Comment:
My advise is not to merge master in your own branch. See the result 2 lines under, MIN and MAX are now duplicated. Try to learn `rebase`. Meaning rebase your branch with master and remove the last merge commit.

Reply:
OK, i will do that.